Sources/Models/NSConstraintLayoutSet.swift

```diff
@@ -59,7 +59,7 @@ public class NSLayoutConstraintSet {
     ///
     /// - Returns: Self
     @discardableResult
-    func activate() -> Self {
+    public func activate() -> Self {
```

Comment:
To be honest, this class should not be public. MessageKit is not a constraint library. What to do about this considering it accidentally got exposed?

Reply:
No exposed currently, so let's make `internal`.